python/src/spark_rapids_ml/core.py

```diff
@@ -471,6 +474,12 @@ def _train_udf(pdf_iter: Iterator[pd.DataFrame]) -> pd.DataFrame:
 
         return pipelined_rdd
 
+    def _fit_array_order(self) -> _ArrayOrder:
```

Comment:
Can we merge _fit_array_order and _transform_array_order into _array_order and move into _CumlCommon?

Reply:
Then it would need to be overridden in both Model and Estimator which kind of defeats the purpose.     I don't see an algo specific common class that subclasses _CumlCommon .   I think will leave as is for now.